tensorbay/opendataset/nuImages/loader.py

```diff
@@ -110,7 +110,11 @@ def _get_annotation_info(info_path: str, is_test: bool = False) -> Dict[str, Any
             info_path, "surface_ann", no_key_frame=True, determined_token="sample_data_token"
         )
         annotation_info["category"] = get_info_with_token(info_path, "category")
-        annotation_info["attribute"] = get_info_with_token(info_path, "attribute")
+        attribute_details = get_info_with_token(info_path, "attribute")
```

Comment:
what is `info` what is `detail` what is the difference?

`attribute_details = get_info_with_token(info_path, "attribute")`
why we get a `details` from a function named `get_info`?